### PR TITLE
AUTH-126: DBP Postman tests refactoring

### PR DESCRIPTION
**Pull request by kconkas**

### CODE REVIEW

File: test/postman/dbp-test-collection.json [hunk 1]
```diff
@@ -900,5 +873,91 @@
...
+			"id": "aaf6ca65-cd8a-44be-881f-b5fa35fdf4d0",
+			"key": "identityServiceUrl",
+			"value": "{{url}}/auth",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "e32b0a99-5c6a-40b4-9e6b-ba95340ceab1",
+			"key": "identityServiceRealm",
+			"value": "springboot",
```
Comment (cristina-sirbu) [inline test/postman/dbp-test-collection.json]: By default shouldn't be alfresco?

File: test/postman/dbp-test-collection.json [hunk 2]
```diff
@@ -900,5 +873,91 @@
...
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "f30db13c-769a-4fc0-9230-cfcfb3463818",
+			"key": "identityServiceClientId",
+			"value": "activiti",
```
Comment (cristina-sirbu) [inline test/postman/dbp-test-collection.json]: By default shouldn't be alfresco?

File: test/postman/dbp-test-collection.json [hunk 3]
```diff
@@ -900,5 +873,91 @@
...
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "b87f9762-2628-4c8e-8700-b5c726116f0c",
+			"key": "testUserUsername",
+			"value": "hruser",
```
Comment (cristina-sirbu) [inline test/postman/dbp-test-collection.json]: By default shouldn't be the user that's in alfresco realm?

File: test/postman/dbp-test-collection.json (hunk 1 above)
Comment (kconkas) [reply test/postman/dbp-test-collection.json]: Thanks for catching this, I've pushed a new commit that addresses this.

File: test/postman/dbp-test-collection.json (hunk 2 above)
Comment (kconkas) [reply test/postman/dbp-test-collection.json]: Thanks for catching this, I've pushed a new commit that addresses this.

File: test/postman/dbp-test-collection.json (hunk 3 above)
Comment (kconkas) [reply test/postman/dbp-test-collection.json]: Thanks for catching this, I've pushed a new commit that addresses this.

**Pull request merged**